addons/arsenal/arsenale/fnc_arsenalBW.sqf

```diff
@@ -496,37 +534,11 @@ _items append (switch (ACE_player getVariable ["TB_rolle", ""]) do
 
             // Weste
             "BWA3_Vest_Grenadier_Fleck",
-            "BWA3_Vest_Grenadier_Tropen"
-        ]
-    };
-
-    case "dmr":
```

Comment:
soll die DMR Rolle raus? dann muss hier aber was leeres zurück
`case "dmr": {[]};` (wie in USA) zudem muss die Rolle geblacklistet werden, mache ich dann wenn es so gewünscht ist